packages/chart/src/FigureChartModel.ts

```diff
@@ -72,17 +72,13 @@ class FigureChartModel extends ChartModel {
     this.settings = settings;
     this.theme = theme;
     this.data = [];
-    const template = {
-      data: {},
-      layout: this.chartUtils.makeDefaultLayout(theme),
-    };
     this.layout = {
       grid: {
         rows: figure.rows,
         columns: figure.cols,
         pattern: 'independent',
       },
-      template,
+      template: this.chartUtils.makeDefaultTemplate(theme),
```

Comment:
It would be ideal if the model didn't need the theme at all, but looks like it does for some dynamic axis stuff still.

You should be able to remove the template at least because it's overridden by `Chart` anyway

Reply:
I did a bit of testing and you can remove theme from the `updateFigureAxes` call. The only spot where it ends up using it is to set the axis if one doesn't exist, but plotly applies from the template in that case it seems.

Just to test, get a subplotting example from [here](https://deephaven.io/core/docs/how-to-guides/plotting/subplots/) and then change something in `makeLayoutAxis`. I made the y axis `zerolinecolor` red and it properly applied to both series even w/o the assignment in `updateLayoutAxes`

That then leaves just the error bars in the model which I bet can remove the theme as well since you added it to the default template. I couldn't find a good example on the docs to test this though